src/models/wc.js

```diff
@@ -0,0 +1,19 @@
+import { round } from "../utilities/utilities.js";
+
+/**
+ * Calculates the Wind Chill Index (WCI) in accordance with the ASHRAE 2017 Handbook Fundamentals - Chapter 9 [18]_.
+ * @param {number} tdb - dry bulb air temperature,[°C]
+ * @param {number} v - wind speed 10m above ground level, [m/s]
+ * @param {boolean} ifRound - default True, if True rounds output value, if False it does not round it
+ * @returns {object} {"wci": wci} - wind chill index, [W/m2)]
+ */
+
+export function wc(tdb, v, ifRound = true) {
```

Comment:
can you take kwargs and call the option round? to be consistent with the rest of the functions.
something like:
```javascript
function wc(tdb, v, {round} = {round: true}) {
}
```